### DISCUSSION TITLE
Is there any plan to support alpine linux,thanks

### DISCUSSION THREAD

**Discussion by tomyouyou** [rabbitmq/rabbitmq-server#2431] [Other] [answered]

**Comment by michaelklishin** [comment, marked as answer]
There isn't anything to specifically support, as the [community Docker image demonstrates](https://github.com/docker-library/rabbitmq/tree/master/3.8/alpine). In fact, for that image, we have seen suggestions to drop support for Alpine and continue with just Debian/Ubuntu.

**Comment by michaelklishin** [comment]
To be clear, there are no plans [known to me] to drop support for Alpine Linux in the image. However, it tells me that there isn't enough interest to warrant e.g. production of Alpine-specific packages from the core team.

**Comment by gerhard** [comment]
@tomyouyou in what way? Do you mean Alpine Linux package?
